pandas/tests/scalar/period/test_period.py

```diff
@@ -1187,6 +1187,15 @@ def test_comparison_numpy_zerodim_arr(self, zerodim_arr, expected):
 
 
 class TestArithmetic:
+    def test_add_sub_td64_nat(self):
+        # GH#47196
+        per = Period("2022-06-01", "D")
+        nat = np.timedelta64("NaT", "ns")
+
+        assert per + nat is NaT
+        assert nat + per is NaT
+        assert per - nat is NaT
```

Comment:
`nat - per` goes through `NaT` add/sub logic?

Reply:
not quite, it raises.  ill add that to this test

Reply:
Hmm shouldn't `nat - per` also be `NaT` instead of raise?

Reply:
this is specifically a timedelta64 nat, so no

Reply:
Ah right it's a numpy object. Makes sense.